Gifski/ConversionCompletedViewController.swift

```diff
@@ -70,6 +70,10 @@ final class ConversionCompletedViewController: NSViewController {
 				return
 			}
 
+			guard self.view.window?.contentViewController == self else {
```

Comment:
I feel like this is just hiding a different issue. `self` is already weak at this point, so the only way this can happen is if this controller is retained while not being the current one.

Reply:
This is true. There seems to be some time during the transition that this can be invoked but before ARC has cleanup up the popped VC

Reply:
If there's really no better solution than this, it should at least have a code comment about this.

Reply:
I'm not sure why viewDidAppear is getting called at all on this view controller after clicking back. Will do some more digging